src/components/PersonalSettings.vue

```diff
@@ -13,12 +13,12 @@
 			<br><br>
 			<div v-if="connected" class="openproject-grid-form">
 				<label class="openproject-connected">
-					<a class="icon icon-checkmark-color" />
+					<CheckIcon class="openproject-connected-checkmark" :size="20" fill-color="#007500" />
```

Comment:
You can set a green color with the "color" css attribute on material icons. There is a `--color-success` CSS variable provided by Nextcloud.

```suggestion
					<CheckIcon class="openproject-connected-checkmark" :size="20" />
```

Reply:
Done in the latest commit. Thank you for the suggestion!